libloki/crypto.js

```
@@ -48,6 +48,12 @@
     // Should we use ephemeral key pairs here rather than long term keys on each side?
     async encrypt(plaintext) {
       const myKeyPair = await textsecure.storage.protocol.getIdentityKeyPair();
+      if (!myKeyPair) {
+        window.log.warn("encrypt, Can't load myKeyPair from storage");
+        // FIXME: not sure what I should give on failure
+        // just going to stick with the same type of value
+        return {};
```

We should be throwing an error instead of returning empty value.